spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1720,143 +1717,15 @@ public void testInvalidAPIUsage() {
 
     assertThatThrownBy(() -> actions().rewriteDataFiles(table).binPack().sort())
         .isInstanceOf(IllegalArgumentException.class)
-        .hasMessage("Must use only one rewriter type (bin-pack, sort, zorder)");
+        .hasMessage("Must use only one runner type (bin-pack, sort, zorder)");
```

Comment:
I'm not sure about this change.
The internal naming has changed, but not sure we have to expose this to the users

Reply:
I wouldn't go with the "runner" term.  That's not really a concept in Spark, so I think this just exposes the internals in a confusing way.

Reply:
We use "strategy" in the [user doc](https://iceberg.apache.org/docs/nightly/spark-procedures/#usage_9).

Reply:
I decided to keep this error message intact.